src/API/Base.php

```diff
@@ -236,9 +236,7 @@ protected static function parse_response( $response ) {
 			throw new \Exception( __( 'Empty body', 'pinterest-for-woocommerce' ), 204 );
 		}
 
-		$body = (array) json_decode( $response['body'] );
-
-		return $body;
+		return (array) json_decode( $response['body'] );
```

Comment:
Minor: Instead of casting to an array, you can pass the second parameter to `json_decode()` to get an array back.

```suggestion
		return json_decode( $response['body'], true );
```

Reply:
The end result is not same. This would require more changes as we are expecting objects as members of that array in multiple places.